### PR TITLE
FW-606  Remove Jquery dependency from chat.min.js file.

### PR DESCRIPTION
**Pull request by ab14bhardwaj**
### What do you want to achieve?
- Remove Jquery dependency from chat.min.js file.
- Add alternative JS function for jquery .extend.
- Remove applozic.calling.js file from chat.min.js file array.

### How was the code tested?

- By running the node server and checking the basic functionality.

### What new thing you came across while writing this code? 
- http://youmightnotneedjquery.com/

### In case you fixed a bug then please describe the root cause of it? 
-> Widget code was not working in IE because of "Object.assign" 

NOTE: Make sure you're comparing your branch with the correct base branch

### CODE REVIEW

File: public/plugin/js/app/modules/group/applozic.group.js [hunk 1]
```diff
@@ -81,11 +81,12 @@ function MckGroupUtils() {
   _this.addGroup = function (group) {
     var name = (group.name) ? group.name : group.id;
     var users = [];
-    $applozic.each(group.groupUsers, function (i, user) {
+    group.groupUsers.forEach(function(user, i){
```
Comment (devashishmamgain) [inline public/plugin/js/app/modules/group/applozic.group.js]: see if we need undefined check for group.groupUsers

File: public/plugin/js/app/modules/group/applozic.group.js [hunk 2]
```diff
@@ -236,7 +237,7 @@ function MckGroupService() {
 
   _this.init = function (optns) {
     IS_MCK_VISITOR = optns.visitor;
-    MCK_USER_ID = (IS_MCK_VISITOR) ? 'guest' : $applozic.trim(optns.userId);
+    MCK_USER_ID = (IS_MCK_VISITOR) ? 'guest' : optns.userId.trim();
```
Comment (devashishmamgain) [inline public/plugin/js/app/modules/group/applozic.group.js]: see if undefined check is required

File: public/plugin/js/app/modules/group/applozic.group.js (hunk 1 above)
Comment (ab14bhardwaj) [reply public/plugin/js/app/modules/group/applozic.group.js]: Fixed.

File: public/plugin/js/app/modules/group/applozic.group.js (hunk 2 above)
Comment (ab14bhardwaj) [reply public/plugin/js/app/modules/group/applozic.group.js]: Fixed.

File: public/plugin/js/app/modules/group/applozic.group.js [hunk 3]
```diff
@@ -584,20 +585,20 @@ function MckGroupService() {
...
       if (params.groupId) {
-        messagePxy.groupId = $applozic.trim(params.groupId);
+        messagePxy.groupId = params.groupId.trim();
       } else if (params.clientGroupId) {
         var group = mckGroupUtils.getGroupByClientGroupId(params.clientGroupId);
         if (typeof group === 'undefined') {
           return 'group not found';
         }
-        messagePxy.clientGroupId = $applozic.trim(params.clientGroupId);
+        messagePxy.clientGroupId = params.clientGroupId.trim();
```
Comment (devashishmamgain) [inline public/plugin/js/app/modules/group/applozic.group.js]: not sure if this will be always present

**Pull request merged**